osmosis-1/osmosis.zone.json

```diff
@@ -1350,6 +1350,16 @@
       "osmosis_main": false,
       "osmosis_frontier": true,
       "osmosis_info": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1u45shlp0q4gcckvsj06ss4xuvsu0z24a0d0vr9ce6r24pht4e5xq7q995n",
+      "pools": {
+        "OSMO": 894
+      },
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "osmosis_info": true
```

Comment:
please make osmosis_info false. There is not enough liquidity to qualify for tracking (min $1000)